### ISSUE TITLE
Resource/course updating successfully though there is no change

### ISSUE THREAD

**Issue by BhargaviNadendla** [open-learning-exchange/planet#3346]
Though if there are no changes made on update and the form has been untouched, it gives a message that course or resource updated succesfully. Do we need to avoid the `post` request?

![image](https://user-images.githubusercontent.com/25775298/53967528-06e14780-40bb-11e9-88ac-ec732e990b29.png)

**Comment by paulbert** [comment]
Yes, we could do that to save some bandwidth & reduce the number of times the revision number increases, but this is lower priority.
